erc20/src/lib.rs

```diff
@@ -93,7 +95,11 @@ impl ERC20 {
 
         let to_balance = self.balances.read(to);
         self.balances.write(to, to_balance + value);
-        log::emit(Transfer::new(0, to, value));
+        log::emit(Transfer::new(
+            address!("0000000000000000000000000000000000000000"),
```

Comment:
Why is `alloy` not expecting `[u8; 32]` here? Then `0` would have been a compile error.

Reply:
```
error[E0308]: mismatched types
  --> src/lib.rs:96:33
   |
96 |         log::emit(Transfer::new(0, to, value));
   |                   ------------- ^ expected `Address`, found integer
```
@deuszx I'm not sure if I understood the question correctly, but I see compiler error as above.